src/geopathfinder/naming_conventions/yeoda_naming.py

```diff
@@ -299,21 +299,18 @@ def yeoda_path(root, product=None, version=None, run_num=None, datalog='datasets
     else:
         raise ValueError('Wrong input for "datalog" level!')
 
-    # define the version and run number folder name
-    if version:
-        if isinstance(version, int):
-            version = 'V' + "{:02d}".format(version)
-    if run_num:
-        run_num = 'R' + "{:02d}".format(run_num)
-    wflow = str(version) if version else ""
-    wflow += str(run_num) if run_num else ""
-    wflow = wflow if version or run_num else None
+    # define the data_version and run number folder name
+    if data_version:
```

Comment:
If you agree I would change that to `if data_version is not None` to allow zero as an input.

Reply:
done